EzMap.Domain/Models/EzMapContext.cs

```diff
@@ -8,6 +9,10 @@ public class EzMapContext : DbContext
     
     public DbSet<User> Users => Set<User>();
     
+    public DbSet<PoiCollection> PoiCollections=> Set<PoiCollection>();
+
+    public DbSet<Tag> Tags => Set<Tag>();
```

Comment:
you just need these db set to be public property, it's gonna be populated by ef anyways, you'll never touch it.
`public DbSet<Tag> Tags => {get; set;}`